cmd/root.go

```diff
@@ -81,6 +103,22 @@ var rootCmd = &cobra.Command{
 		os.Exit(1)
 	},
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		//verify output option
+		if !stringInSlice(outputFormat, outputOptions) {
+			return fmt.Errorf("--output must be one of %v", outputOptions)
+		}
+
+		if outputFormat == "custom" {
+			if len(customColumns) < 1 {
+				return fmt.Errorf("when --output=custom you must specify --columns")
+			}
```

Comment:
```suggestion
			}
			// Uppercase all columns entered on CLI
			var tempColumns []string
			for _, colString := range customColumns {
				tempColumns = append(tempColumns, strings.ToUpper(colString))
			}
			customColumns = tempColumns
```
Disclaimer: accepting this suggestion will break the build, because you'll need to add `"strings"` to the import list at the top of the file.

If we do something like this, you can enter whatever case you want on the command line. Ex.
`pluto detect-helm -ocustom --columns=nAmE`

Reply:
great idea. dONe